Description: Use gnome-session's builtin session manager
 As it stands, when using the systemd session manager, gnome-shell will be
 started indiscriminately. This breaks budgie-wm. This patch makes budgie use
 gnome-session's builtin session manager.
Author: Balló György <ballogyor+arch at gmail dot com>
Origin: vendor, https://git.archlinux.org/svntogit/community.git/tree/trunk/PKGBUILD?h=packages/budgie-desktop
Bug-Archlinux: https://bugs.archlinux.org/task/63849
---
This patch header follows DEP-3: http://dep.debian.net/deps/dep3/
--- a/src/session/budgie-desktop.in
+++ b/src/session/budgie-desktop.in
@@ -13,4 +13,4 @@
   export XDG_CURRENT_DESKTOP
 fi
 
-exec gnome-session --session=budgie-desktop $*
+exec gnome-session --builtin --session=budgie-desktop $*
